src/interface_adapter/display_requests/DisplayRequestsController.java

```diff
@@ -0,0 +1,17 @@
+package interface_adapter.display_requests;
+
+import entity.User;
+import use_case.display_requests.DisplayRequestsInputBoundary;
+
+public class DisplayRequestsController {
+    final DisplayRequestsInputBoundary displayRequestsInteractor;
+    public DisplayRequestsController(DisplayRequestsInputBoundary displayRequestsInteractor) {
+        this.displayRequestsInteractor = displayRequestsInteractor;
+    }
+
+    public void execute() {
+        User user = LoggedInState.getCurrentUser();
+        displayRequestsInteractor.execute(user);
```

Comment:
Our current design is to pass in username instead of User object into the controller and interactor.